ts/components/inputs/SessionInput.tsx

```diff
@@ -386,11 +404,29 @@ export const SessionInput = (props: Props) => {
       >
         {isTextArea ? (
           <StyledTextAreaContainer {...containerProps}>
-            <textarea
-              {...inputProps}
-              ref={inputRef || textAreaRef}
-              aria-label={ariaLabel || 'session input text area'}
-            />
+            {isFocused ? (
+              <textarea
+                {...inputProps}
+                placeholder={''}
+                ref={inputRef || textAreaRef}
+                aria-label={ariaLabel || 'session input text area'}
+              />
+            ) : (
+              <StyledPlaceholder
+                error={textErrorStyle}
+                textSize={textSize}
+                editable={editable}
+                centerText={centerText}
+                monospaced={monospaced}
+                onClick={() => {
+                  if (editable) {
+                    setIsFocused(true);
+                  }
+                }}
```

Comment:
oh this is not pretty

Reply:
we could have done the same with a
StyledTextAreaContainer: {relative}
textarea: {absolute, top = bottom = left = right = 0}
placeholder: {absolute, click-events: none, top = bottom = left = right = 0}

but we are way too far

Reply:
Yeah it would still require logic for centering the placeholder text within the relative height of the textarea. But I agree with being too far.

See code on the left:

```
position: absolute;
    top: ${props =>
      `calc(var(--font-size-${props.textSize}) + ${props.textSize === 'xl' ? '8px' : '5px'})`};
```

Reply:
Actually unsure if by doing `position: absolute; inset: 0;` we would we get centered text 🤔